screens/Budget/components/missingBudget.tsx

```diff
@@ -73,7 +87,7 @@ const MissingBudget = (
             />
             <Button
                 title="Copy Last Month's Budget"
```

Comment:
Maybe rename this to "Copy most recent budget", because that is what the logic actually does.